3-enrich/scala-common-enrich/project/Dependencies.scala

```diff
@@ -50,6 +50,7 @@ object Dependencies {
     val iabClient        = "0.1.0"
     val yauaa            = "5.8"
     val kryo             = "2.24.0"
+    val guava            = "28.1-jre"
```

Comment:
Can we mention that direct dependency is only used for IP representation (just in case somebody will find a better tool for that a decide to get rid of it entirely)?

Reply:
comment added